tools/scripts/buildMission.bat

```diff
@@ -0,0 +1,8 @@
+REM ~dp0 ist die Variable, die auf den Ordner der Batch-Datei verweist. So kann die Datei von überall aufgerufen werden, ohne dass sich relative Pfade ändern.
+REM ~dp0 muss in "" eingefasst werden, sonst gibt es Probleme, wenn Leer- oder Sonderzeichen im Pfad enthalten sind. Yay, Windows!
+
+REM TODO: -x glob, um .git und tools auszuschließen
+REM TODO: Konfig, um Arma 3\MPMissions-Dir angeben zu können, wohin die Mission gepackt werden soll, z.B. "E:\Steam\steamapps\common\Arma 3\MPMissions\OPT_Dev.Altis.pbo"
```

Comment:
Da könntest du eine ähnliche Lösung verwenden, wie ich sie für das Server-Startskript gewählt habe: Ein extra "Config-Skript" das entsprechende Variablen setzt. Und da es in batch keine lokalen Variablen gibt, existieren die Variablen dann im aufrufenden  Skript nachdem das "Config-Skript" via `call` aufgerufen wurde.

https://github.com/Krzmbrzl/OPT-Mod/blob/serverStartSkript/tools/scripts/server/setMetaData.bat.example

Reply:
Hoffe ehrlich gesagt, dass sich jemand anderes des Skripts annimmt :) . Werde da mal form anhauen, der wollte sich ja gern um so Zeug kümmern. Dass das Skript hinten und vorne nicht passt, ist mir klar. @formtapez

Reply:
Ja dann denke ich, würde es Sinn machen das Skript ganz aus dem PR zu nehmen. Halbfertige Sachen sollten nicht gemerged werden. Sonst geraten sie in Vergessenheit und niemand kümmert sich anständig drum xD

Reply:
Ich habe in der Richtung mal ein bisschen rumprobiert und ein bisschen was vorbereitet, habe aber massive Probleme:

- Mit "armake2 build ..." scheitert es ja leider an den fehlerhaft erkannten Kommentaren, da hat Raven ja schon ein bugfix gepullrequested - davon gibts aber noch gar kein build?
- Mit "armake2 pack ..." kann ich die Mission packen, aber der --exclude Parameter scheint absolut überhaupt nicht zu fruchten. Ist euch da was bekannt?

Reply:
Wenn du den Build aus dem mod-Repo nimmst, dann hast du eine Version, die den Fix bezüglich der Kommentare schon enthält.

Dass `pack` den exclude ignoriert, istmir neu. Da erstellst du am Besten n Issue im armake2-repo: https://github.com/KoffeinFlummi/armake2/issues

Reply:
Kann ich so nicht bestätigen.
Das armake2 aus dem mod-repo ist das einzige was ich habe, und das failed bei der empty mission so:
`  #include "dialogs\include.hpp" // Include dialog configs
                                 ^
Unexpected token "/", expected: "[ \t]", "\n", "\r\n"`

Reply:
Okay - das werde ich mal testen...

Reply:
Grad getestet: Es geht auch bei mir nicht. Dann werde ich wohl nochmal in den Code schauen müssen xD